lib/services/android-project-service.ts

```diff
@@ -133,18 +133,32 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		return Promise.resolve(true);
 	}
 
-	public getAppResourcesDestinationDirectoryPath(projectData: IProjectData, frameworkVersion?: string): string {
-		if (this.canUseGradle(projectData, frameworkVersion)) {
-			const resourcePath: string[] = [constants.SRC_DIR, constants.MAIN_DIR, constants.RESOURCES_DIR];
-			if (this.isAndroidStudioTemplate) {
-				resourcePath.unshift(constants.APP_FOLDER_NAME);
-			}
+	public getAppResourcesDestinationDirectoryPath(projectData: IProjectData): string {
+		const appResourcesDirStructureHasMigrated = this.$androidResourcesV4MigrationService.hasMigrated(projectData.getAppResourcesDirectoryPath());
+
+		if (appResourcesDirStructureHasMigrated) {
+			return this.getAppResourcesDestinationDirectoryPathUpdatedAppResourcesDirStructure(projectData);
+		} else {
+			return this.getAppResourcesDestinationDirectoryPathOldAppResourcesDirStructure(projectData);
+		}
+	}
+
+	private getAppResourcesDestinationDirectoryPathOldAppResourcesDirStructure(projectData: IProjectData): string {
+		const resourcePath: string[] = [constants.SRC_DIR, constants.MAIN_DIR, constants.RESOURCES_DIR];
+		if (this.isAndroidStudioTemplate) {
```

Comment:
Hm, I have some concerns about this property. The `androidProjectService` is a singleton and in case you are using CLI as a library (in Sidekick), you may work with different type of projects. However, once the value of `this.isAndroidStudioTemplate` is set to true, it is never reset to false (in case the currently opened project is not from Android studio compatible template). This may cause some issue, but we can discuss them offline and see how to resolve them